### DISCUSSION TITLE
Template Schema Options

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3078] [Feature Requests]
The easiest way to get started with Directus. These would include simple, starter schemas for keystone project types. They could also include specialized Extensions (interfaces, layouts, displays, and modules) that improve the experience. These should be selectable upon installation (or "Blank")... and should probably pull from a centralized server so they can be updated and consolidated as a marketplace.

* Portfolio Website
* Project Management
* Agency Framework
* Product Roadmap
* Support Ticket System
* Customer Relationship Management

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
